node/blockchain/models/lock.py

```diff
@@ -0,0 +1,7 @@
+from djongo import models
+from djongo.models import ObjectIdField
+
+
+class Lock(models.Model):
+    _id = ObjectIdField()
+    name = models.CharField(max_length=63, unique=True)
```

Comment:
why 63?

Reply:
It's really strange. Look like my hand trembled.